tests/two_d_primitive_tests.h

```diff
@@ -0,0 +1,35 @@
+#ifndef NXDK_PGRAPH_TESTS_2D_PRIMITIVE_TESTS_H
+#define NXDK_PGRAPH_TESTS_2D_PRIMITIVE_TESTS_H
+
+#include <pbkit/pbkit.h>
+
+#include "test_suite.h"
+
+class TestHost;
+
+class TwoDPrimitiveTests : public TestSuite {
+ public:
+  struct TwoDPrimitiveTest {
+    uint32_t two_d_object_color;
+    uint32_t two_d_start_x;
+    uint32_t two_d_start_y;
+    uint32_t two_d_end_x;
+    uint32_t two_d_end_y;
+  };
+
+ public:
+  TwoDPrimitiveTests(TestHost& host, std::string output_dir);
+
+  std::string Name() override { return "2D Primitive"; }
```

Comment:
If you plan on doing more than just lines in this suite I think this name is fine, but if you want to test other primitives elsewhere maybe just "2D Lines"?

Reply:
Originally I thought I would be able to do rectangle's and others but I never saw those exposed in the header files.  I think for now I'll convert to LINES (although I'm not sure yet if we're creating lines in hardware at all or just LIN's ;)

Reply:
done